src/bluez/adapter.rs

```diff
@@ -101,6 +101,12 @@ impl Central for Adapter {
         ))
     }
 
+    async fn clear_peripherals(&self) -> Result<()> {
+        Err(Error::NotSupported(
```

Comment:
I'm worried that this will result in people writing programs that crash on Linux as they only tested it on other platforms. What do you think about this just being a silent no-op on Linux? Not being able to clear the peripheral list shouldn't be a big issue in most cases I can think of.

Reply:
That sounds good, I'll change it. I think it may be possible to clear the bluez list, but the bluez-async lib doesn't seem to currently support it.